src/main/java/org/scijava/plugins/scripting/python/OptionsPython.java

```diff
@@ -208,6 +282,50 @@ public void save() {
 		}
 	}
 
+	private File writeEnvironmentYaml() {
+		File envFile = getEnvironmentYamlFile();
+
+		// skip writing if nothing has changed
+		if (initialCondaDependencies.equals(condaDependencies) &&
+			initialPipDependencies.equals(pipDependencies)) return envFile;
+
+		// Update initial dependencies to detect future changes
+		initialCondaDependencies = condaDependencies;
+		initialPipDependencies = pipDependencies;
+
+		// Write environment.yml from condaDependencies and pipDependencies
+		try {
+			String name = "fiji";
+			String[] channels = { "conda-forge" };
+			String pyimagej = "pyimagej>=1.7.0";
+			String apposePython =
+				"git+https://github.com/apposed/appose-python.git@efe6dadb2242ca45820fcbb7aeea2096f99f9cb2";
```

Comment:
We should not hardcode versions in Java. Can we instead parse them out of the existing shipped `environment.yml` when populating the GUI, so they will propagate naturally?